### PR TITLE
Make Intl localization default in DevExtreme

### PR DESCRIPTION
**Pull request by nightskylark**
See first commit  to exclude ES6 converting changes.

### CODE REVIEW

**Comment by nightskylark** [comment]
@dimarudnev, @AlekseyMartynov, @RomanTsukanov FYI.

File: js/localization/date.js [hunk 1]
```diff
@@ -90,12 +92,9 @@ var dateLocalization = dependencyInjector({
     },
 
     is24HourFormat: function(format) {
-        var amTime = new Date(2017, 0, 20, 11, 0, 0, 0),
-            pmTime = new Date(2017, 0, 20, 23, 0, 0, 0),
-            amTimeFormatted = this.format(amTime, format),
-            pmTimeFormatted = this.format(pmTime, format);
+        const amTime = new Date(2017, 0, 20, 11, 0, 0, 0), pmTime = new Date(2017, 0, 20, 23, 0, 0, 0), amTimeFormatted = this.format(amTime, format), pmTimeFormatted = this.format(pmTime, format);
```
Comment (tar-g) [inline js/localization/date.js]: Line too long

File: js/localization/default_date_names.js [hunk 2]
```diff
@@ -1,19 +1,16 @@
-var iteratorUtils = require("../core/utils/iterator");
+import iteratorUtils from "../core/utils/iterator";
 
-var MONTHS = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
-    DAYS = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"],
-    PERIODS = ["AM", "PM"],
-    QUARTERS = ["Q1", "Q2", "Q3", "Q4"];
+const MONTHS = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"], DAYS = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"], PERIODS = ["AM", "PM"], QUARTERS = ["Q1", "Q2", "Q3", "Q4"];
```
Comment (tar-g) [inline js/localization/default_date_names.js]: IMO previous formatting was better

**Review by tar-g — DISMISSED** [review]

**Comment by AlekseyMartynov** [comment]
I'm interested whether this induces changes to DevExtreme-based ASP.NET project templates and [Localization docs](https://docs.devexpress.com/AspNetCore/400577/).

- Will we need to exclude CLDR and Globalize from default included scripts?
- Will we need to mark `Intl` as a better choice in docs?

cc @MargaritaLoseva

File: js/localization/default_date_names.js (hunk 2 above)
Comment (nightskylark) [reply js/localization/default_date_names.js]: It was not perfect lebab config. I'll fix.

**Comment by nightskylark** [comment]
@AlekseyMartynov Intl and Globalizae has its own pros and cons. But Intl is easier to setup, so it's better to start with it if you are not sure what to choose.

**Comment by AlekseyMartynov** [comment]
We had a call with @nightskylark, and now I understand what we need to do. Thanks.

**Review by tar-g — APPROVED** [review]

**Pull request merged**